shacl12-node-expr/index.html

```diff
@@ -618,6 +608,42 @@ <h3>Literal Expressions</h3>
 
 			</section>
 
+			<section id="blank-node-functions">
+				<h2>Node Expressions based on Blank Nodes</h2>
+				<p>
+					...
+				</p>
+				<section id="ListParameterFunction">
+					<h3>List Parameter Functions</h3>
+					<p>
+						...
+					</p>
+				</section>
+				<section id="NamedArgumentFunctions">
+					<h3>Named Argument Functions</h3>
+					<p>
+						...
+					</p>
+				</section>
+			</section>
+
+		</section>
+
+		<section id="library">
+			<h2>Node Expressions Library</h2>
+			<p>
+				This section defines all <a>node expression functions</a> that are built into SHACL engines
+				that implement this specification.
+			</p>
+			<p>
+				The syntax definitions of node expression functions that are based on <a>blank nodes</a>
+				typically use a table of properties that these blank nodes can or must have.
+				Such blank nodes are only <a>well-formed</a> if they are not the <a>subject</a> of any other
+				triples, or where any of these properties are used more than once.
```

Comment:
Trying to make this sentence gramatically sensible.
```suggestion
				Such blank nodes are only <a>well-formed</a> when they are not the <a>subject</a> of any other
				triples, and when none of these properties is used more than once.
```